apps/etterlatte-behandling/src/main/kotlin/behandling/revurdering/RevurderingService.kt

```diff
@@ -157,6 +157,10 @@ class RealRevurderingService(
 
         hentBehandling(opprettBehandling.id)
     }?.also { revurdering ->
+        // Ikke oppdater persongrunnlag ved regulering
+        if (prosessType == Prosesstype.MANUELL) {
```

Comment:
Legg på en confirmmocks og verify at den her blir kalt for testene så er jeg happy

Reply:
[61c336c](https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/1741/commits/61c336c67a3a2617f475e07ccffecbfb6497ca6e)